packages/core/src/ItemWithParams/ParamsValueEvaluator.ts

```diff
@@ -0,0 +1,22 @@
+import { Param } from '../Param';
+import { ItemValue } from '../types/ItemValue';
+import { StringableParamEvaluator } from './prepareStringable';
+import { ParamsValueEvaluator } from '../types/ItemWithParams';
+import { RepeatableParamEvaluator } from './prepareRepeatable';
+
+export class ParamEvaluator implements ParamsValueEvaluator<any>{
+  private evaluators: ParamsValueEvaluator<Param>[] = [
+    new StringableParamEvaluator(),
+    new RepeatableParamEvaluator(this),
+  ]
+
+  canEvaluate(param: Param): param is Param {
+    return this.evaluators.some(e => e.canEvaluate(param));
+  }
```

Comment:
Linebreak 💅

Reply:
suggestions really valuable 👍. I've made adjustments to the code accordingly. However, regarding the linebreak, I've made changes as shown in the image below. I hope I've understood your point correctly.

![image](https://github.com/ajthinking/data-story/assets/20497176/9b54d9f8-7459-401d-8908-83e1b0513b28)